.github/workflows/codeql.yml

```diff
@@ -1,68 +0,0 @@
-# For most projects, this workflow file will not need changing; you simply need
```

Comment:
Please do not again put additional non PR related changes ...
Why was it necessary to put this into the main CI?

Reply:
see https://github.com/fomantic/Fomantic-UI/pull/2612#discussion_r1050773751